lib/cli/run-helpers.js

```diff
@@ -282,3 +291,15 @@ exports.validateLegacyPlugin = (opts, pluginType, map = {}) => {
     }
   }
 };
+
+const createExitHandler = ({ exit, passOnFailingTestSuite }) => {
+  return code => {
+    const clampedCode = passOnFailingTestSuite
+      ? 0
+      : Math.min(code, 255);
```

Comment:
[Refactor] ...and since `clampedCode` now is done in `createExitHandler`, there's no more need to have it done in `exitMocha` or `exitMochaLater`. Proposal: let's simplify those two functions?